artcommon/tests/test_assembly.py

```diff
@@ -232,7 +232,8 @@ def test_assembly_basis_time_valid(self):
 """
         self.releases_config = Model(dict_to_model=yaml.safe_load(releases_yml))
         self.assertEqual(
-            assembly_basis_event(self.releases_config, 'foo'), datetime(2021, 1, 1, 0, 0, tzinfo=timezone.utc)
+            assembly_basis_event(releases_config=self.releases_config, assembly='foo', build_system='konflux'),
+            datetime(2021, 1, 1, 0, 0, tzinfo=timezone.utc),
```

Comment:
i think we should write a couple of simple tests where there is brew event and time both defined and we assert the right one is fetched based on build system